mantidproject/mantid
# Change of log name Filename to raw_filename in LoadRaw3

Fixes #13648

Previously LoadRaw3 added the original filename in the log as `Filename` this has been updated to `raw_filename` for clarity and to be more in keeping with Mantid log style.  
# To Test
- Open the dialogue for LoadRaw in the algorithms tab in Mantid
- Use any relevant `.Raw` file for the input
- Run LoadRaw
- Ensure that the Sample log contains the entry `raw_filename` with the correct file path to the input file as the value
